docs/source/weightvec.rst

```diff
@@ -74,10 +74,57 @@ vs
 
 :math:`s^2 = \frac{1}{n} \sum_{i=1}^n {\left({x_i - m}\right)^2 }`
 
-The unbiased estimate of the population variance, standard deviation or covariance is computed by replacing :math:`\frac{1}{\sum{w}}` with a factor dependent on the type of weights used:
+However, unbiased estimates (when ``corrected=true``) are dependent on the types of weights used. All weights presented here are a subtype of ``AbstractWeights``.
 
-- ``AnalyticWeights``: :math:`\frac{1}{\sum w - \sum {w^2} / \sum w}`
-- ``FrequencyWeights``: :math:`\frac{1}{\sum{w} - 1}`
-- ``ProbabilityWeights``: :math:`\frac{n}{(n - 1) \sum w}` where ``n`` equals `count(!iszero, w)`
+Weights
```

Comment:
There's also a section above which introduces generic weights without mentioning the other types of weights. It would be good to adapt it, since typically people should use a specific type rather than the generic one.